docs/gaia/gaiacli.md

```diff
@@ -128,6 +128,33 @@ gaiacli keys show --multisig-threshold K name1 name2 name3 [...]
 For more information regarding how to generate, sign and broadcast transactions with a
 multi signature account see [Multisig Transactions](#multisig-transactions).
 
+### Fees & Gas
+
+Each transaction may either supply fees or gas prices, but not both. Most transactions
+will typically provide fees as this is the cost you will end up incurring for
```

Comment:
Maybe "Most users will submit transactions with `--fees` as this is the cost transactions will incur for inclusion in the ledger."

Reply:
I updated to users...but not `--fees` because that assumes a CLI client.